lib/sequent/migrations/view_schema.rb

```diff
@@ -137,6 +137,16 @@ def create_view_tables
               statements = sql_file_to_statements(indexes_file_name) { |raw_sql| raw_sql.remove('%SUFFIX%') }
               statements.each(&method(:exec_sql))
             end
+
+            Dir.glob("#{Sequent.configuration.migration_sql_files_directory}/#{table.table_name}*.sql").each do |file_name|
```

Comment:
How sequent currently works is when creating an alter migration you should update the `table_name.sql` and or `table_name.indexes.sql` with the altered statements manually. I see this is missing in the documentation.